.github/workflows/e2e-test.yml

```diff
@@ -0,0 +1,60 @@
+name: E2E-Test
+on:
+  workflow_call:
+env:
+  GO_VERSION: "~1.18"
+  CONTROLLER_TOOLS_VERSION: "v0.9.2"
+  ENVTEST_K8S_VERSION: "1.24.2"
+  SCHEDULER_COMPATIBLE_K8S_VERSION: "v0.24.3"
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  run-e2e-test:
+    name: Run E2E Tests
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        config:
+          - name: "keptn-lifecycle-operator"
+            folder: "operator/"
+          - name: "scheduler"
+            folder: "scheduler/"
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v3
+        with:
+          go-version: ${{ env.GO_VERSION }}
+          cache: true
+          cache-dependency-path: '**/go.sum'
+
+      - name: Download manifests
+        uses: actions/download-artifact@v3
+        with:
+          path: ~/download/artifacts
+
+      - name: Create k8s Kind Cluster
+        uses: helm/kind-action@v1.3.0
+  
+      - name: Install cert-manager
+        run: |
+          kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.10.0/cert-manager.yaml
```

Comment:
For better maintainability, I think it makes sense to move the version number into an env variable at the top of the workflow.